README.md

```diff
@@ -58,6 +74,9 @@ export default () => {
 
         // ❓ Optional: limit the depth of your hierarachies
         maxDept: 3
+
+        // ❓ Optional: subarray of referenceTo, when it should not be possible to create new types from all referenceTo types
+        creatableTypes: ['site.page']
```

Comment:
Both `S` and `context` from structure builder must be passed as required config now, they are not optional.

Ie:
```

  deskTool({
      structure: (S, context) => {
        return S.list()
          .title('Content')
          .items([
            ...S.documentTypeListItems(), // or whatever other structure you have
            createDeskHierarchy({
              S,
              context,
              title: 'Main table of contents',
              ///...
```

We need to include this in the README, update the types and add validation in createDeskHierarchy that checks these and emits a error message with steps to fix (since this will typically be the only config change from v2).

We should also add a migration notice about this.